bottles/backend/managers/manager.py

```diff
@@ -1415,11 +1415,8 @@ def delete_bottle(self, config: BottleConfig) -> bool:
         path = ManagerUtils.get_bottle_path(config)
         subprocess.run(["rm", "-rf", path], stdout=subprocess.DEVNULL)
 
-        local_bottles_tmp = self.local_bottles.copy()
-        for b in local_bottles_tmp.values():
-            if b.Path == config.Path:
-                del self.local_bottles[b.Name]
-                break
+        del self.local_bottles[config.Name]
```

Comment:
This is useless, as the `update_bottles` function already resets and populates back this variable.
```suggestion
```

Reply:
https://github.com/cyberphantom52/Bottles/commit/bf20e1894db8466370c66ed4322a22449cf170ce

Done